### DISCUSSION TITLE
Remove Info messages from error list

### DISCUSSION THREAD

**Discussion by JonFP** [codefori/vscode-ibmi#188] [Q&A] [answered]
I'd like to be able to tell the error list to forget the info messages - they are just annoying. Can't find a way to specify that.

**Comment by worksofliam** [comment, marked as answer]
In VS Code, there is a way to filter the error types.

You can disable information messages with the filter: 

![image](https://user-images.githubusercontent.com/3708366/121821121-e1155680-cc64-11eb-9026-bf9862a10561.png)

Or, if you never want to see a specific error code again you can add add that code to the settings:

![image](https://user-images.githubusercontent.com/3708366/121821148-1fab1100-cc65-11eb-97f7-4a26f51a8c70.png)
